src/orders/utils/data.ts

```diff
@@ -463,3 +467,27 @@ export const isAnyAddressEditModalOpen = (uri: string | undefined): boolean =>
     "edit-shipping-address",
     "edit-billing-address",
   ].includes(uri);
+
+const NAME_SEPARATOR = ":";
+const getDiscountNameLabel = (name: string) => {
+  if (name.includes(NAME_SEPARATOR)) {
+    const splited = name.split(NAME_SEPARATOR);
+    return splited[1].trim();
+  }
+
+  return name;
+};
+
+export const getDiscountTypeLabel = (
```

Comment:
it could be a hook that calls `useIntl` so you don't have to pass it, and just focus on displaying proper label